permissions.md

```diff
@@ -29,5 +37,21 @@ You can add a permission by simply doing the command `_perms group <name> add <p
 ### Removing a permission
 You can remove a permission by simply doing the command `_perms group <name> remove <permission>` for example if we want to remove the ability to queue songs from Muted users then we could do `_perms group Muted remove flarebot.play`.
 
-## Group linking
+### Group linking
 Group linking is basically linking a group to a role, this allows for you to add a role to a user __without__ them needing to have the group also added to them. It is a nice way to give a whole bunch of people certain permissions. Let's say we want to assign our `Muted` group to the `Muted` Discord role (which can be created by FlareBot in case of mutes). We can simply do `_perms group Muted link @Muted`, now here we can either mention the muted role or use the role ID (Which can be found in the `_roles` command).
+
+## Users
+Users can be given their own permissions individually from groups! These permissions will **always** overwrite group permissions so keep that in mind when checking issues with a permissions setup. Aside from their own permissions, users can also be assigned groups based on roles or manually adding the roles.
+
+For adding groups with roles simply link the group with a role (As shown above) and the user will have access to that groups permissions without any further actions!
+
+To add or remove groups from a user simply use `_perms user <user> group add|remove <group>` to add or remove a group from a user.
```

Comment:
This doesn't really make much sense

Reply:
Which part?

Reply:
It selected the wrong line - 46
This one doesn't have some annoying duplication in though, "to add or remove groups from a user... to ass or remove a group from a user"
So what does this do? Link right?

Reply:
Adding the roles links yeah